### PR TITLE
Add monitor-client to watch repos list

### PR DESCRIPTION
**Pull request by lluiscampos**
And remove mender-python-client-dev

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -92,6 +92,7 @@ var defaultWatchRepositoriesSync = []string{
 	"devicemonitor",
 	// client
 	"mender-binary-delta",
+	"monitor-client",
```
Comment (merlin-northern) [inline main.go:95]: I hereby request a comment preceding this line:
```c
// Monitoring
```

**Review by merlin-northern — APPROVED** [review]
lgtm, Ole is much better expert here, so I also rely on his review.

File: main.go [hunk 2]
```diff
@@ -105,8 +105,6 @@ var defaultWatchRepositoriesSync = []string{
 	// websites
 	"mender.io",
 	"northern-tech-web",
-	// tmp repo - it is private but will be merged/removed later
-	"mender-python-client-dev",
```
Comment (oleorhagen) [inline main.go:109]: Thx, I saw this the other day, and added it to my TODO list :)

**Review by oleorhagen — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (lluiscampos) [reply main.go:96]: ok

**Pull request merged**